aten/src/ATen/native/native_functions.yaml

```diff
@@ -6200,6 +6200,16 @@
   device_check: NoCheck
   dispatch: {}
 
+- func: _nested_get_min_seqlen(Tensor self) -> int
```

Comment:
I think these need to be symbolic to work with dynamic shapes. TBD exploring this

Reply:
This needs more thought / design work to fully cover dynamic shapes. @YuqingJ mentioned to me that a static max length is being used for now, so I expect the patch to work for FIRST.